node/tessel-export.js

```diff
@@ -952,6 +957,251 @@ Tessel.LED.prototype.read = function(callback) {
   });
 };
 
+Tessel.Wifi = function() {
+  var state = {
+    settings: {},
+    connected: false,
+    busy: false
```

Comment:
I recommend we take this out. It was a workaround for the general shitty-ness of the CC3k wifi chip on Tessel 1 and we don't really need it anymore. I highly doubt anyone actually used it before.